src/main/java/com/zendesk/maxwell/MaxwellReplicator.java

```diff
@@ -93,11 +93,12 @@ public void work() throws Exception {
 		if (row == null)
 			return;
 
-		if (!skipRow(row)) {
+		if ( !bootstrapper.shouldSkip(row) && !isMaxwellRow(row) ) {
 			producer.push(row);
+		} else {
+			bootstrapper.work(row, producer, this);
```

Comment:
hm, good point.  Ok, I think we say that for #222 it's supported to run the replication thread off the slave and keep the schema store on the master (one seemingly popular config), but if you want to disconnect them entirely then bootstrapping won't work.